strings/tips.txt

```diff
@@ -42,47 +42,61 @@ As the Captain, you can purchase a new emergency shuttle using a communications
 As a Scientist, getting drunk just enough will speed up research. Skol!
+As a Scientist, you can use the Destructive Analyzer to unlock new tech nodes.
+As an Explorer, you can use the Newscaster units to message the station if you ever find yourself stranded without your ship.
+As an Explorer, you can find technology disks which will unlock new tech nodes when delievered back to R&D!
 As a Roboticist, keep an ear out for anomaly announcements. If you get your hands on an anomaly core, you can build a Phazon mech!
 As a Roboticist, you can repair your cyborgs with a welding tool. If they have taken burn damage, you can remove their battery, expose the wiring with a screwdriver and replace their wires with a cable coil.
 As a Roboticist, you can reset a cyborg's module by cutting and mending the reset wire with a wire cutter.
+As a Roboticist, you can use a multitool on an assembled endoskeleton to enter debugging mode. Here you can change its name, master AI and a couple of other settings.
 As a Roboticist, you can greatly help out Shaft Miners by building a Firefighter APLU equipped with a hydraulic clamp and plasma cutter. The mech is ash storm proof and can even walk across lava!
 As a Roboticist, you can augment people with cyborg limbs. Augmented limbs can easily be repaired with cables and welders.
 As the AI, you can click on people's names to look at them. This only works if there are cameras that can see them, they aren't wearing agent IDs and aren't using digital camouflage as changelings.
 As the AI, you can quickly open and close doors by holding shift while clicking them, bolt them when holding ctrl, and even shock them while holding alt.
 As the AI, you can take pictures with your camera and upload them to newscasters.
 As a Cyborg, choose your module carefully, as only cutting and mending your reset wire will let you repick it. If possible, refrain from choosing a module until a situation that requires one occurs.
-As a Cyborg, you are immune to most forms of stunning, and excel at almost everything far better than humans. However, flashes can easily stunlock you and you cannot do any precision work as you lack hands.
+As a Cyborg, you are immune to most forms of stunning, and excel at almost everything far better than humans. However, flashes and laser pointers can easily stunlock you and you cannot do any precision work as you lack hands.
 As a Cyborg, you are extremely vulnerable to EMPs as EMPs both stun you and damage you. The ion rifle in the armory or a traitor with an EMP kit can kill you in seconds.
 As a Service Cyborg, your spray can knocks people down. However, it is blocked by masks and glasses.
-As an Engineering Cyborg, you can attach air alarm/fire alarm/APC frames to walls by placing them on the floor and using a screwdriver on them.
+As an Engineering Cyborg, you can attach air alarm/fire alarm/APC/camera frames to walls by placing them on the floor right next to the wall and using a screwdriver on them.
```

Comment:
i had no idea this was a thing, holy shit

Reply:
The problem is that your sprite obscures it, need to alt-click the tile under yourself.